Sources/Benchmark/OperatingSystemStats/OperatingSystemStatsProducer+Darwin.swift

```diff
@@ -117,15 +132,18 @@
                         let quit = self.runState
                         self.lock.unlock()
 
+                        sampleSemaphore.signal()
```

Comment:
Maybe signal only once?

Reply:
Yeah, will fix that separately, thanks!

Reply:
https://github.com/ordo-one/package-benchmark/issues/202